helm-charts/k8s-mediaserver/templates/jackett-resources.yml

```diff
@@ -71,7 +71,7 @@ spec:
                 name: jackett-config
           image: "{{ .Values.jackett.container.image }}:{{ .Values.jackett.container.tag | default .Values.general.image_tag }}"
           imagePullPolicy: Always
-          readinessProbe:
+          readiness probe:
```

Comment:
I'm not sure we want to set this up with a space in the key. It would be different than all the other ones and also this isn't typically what people do since camel case is mostly the norm as far as I can tell.